inst/project-templates/setup-medidata.qmd

```diff
@@ -0,0 +1,74 @@
+---
+title: "{{stringr::str_replace_all(folder_name, stringr::fixed('-'), stringr::fixed(' '))}}"
+subtitle: "Medidata Setup"
+date: "`r Sys.Date()`"
+format:
+  html:
+    toc: true
+    toc-float: true
+    toc-depth: 2
+    toc-location: left
+embed-resources: true
+execute: 
+  echo: true
+---
+
+```{r setup}
+#| include: false
+set.seed({{format(Sys.Date(), format = "%Y%m%d")}})
+options(subdir_name = "{{folder_name}}", openxlsx.dateFormat = "yyyy-mm-dd")
+```
+
+1. Import all raw Medidata datasets into a list
+
+1. Clean each of the Medidata datasets individually as needed. There are no changes to the structure of these data sets in this step, meaning that if a raw data set is one line per person per AE, the cleaned data set will have the same structure.
+
+# Raw Data Import
+
+```{r import}
+library(dsverse); library(tidyverse)
+# library(tidylog)
+
+# setting MSK ggplot theme, gtsummary theme, and color palette
+theme_set(mskRvis::theme_biostat()); theme_gtsummary_pcctc(); mskRvis::set_msk_palette()
+
+# import sas files
+lst_medidata_raw_data <-
```

Comment:
oooh i see you have a separate script for medidata set up. i think i started the project with medidata FALSE to see how it went.  often analyses need a bespoke analysis file & medidata - should this be one script, or kept separate? (and different deliverables will have different bespoke excel files)

Reply:
as discussed, we will keep medidata separate from other setups